src/components/OrgFile/components/AttributedString/index.js

```diff
@@ -1,36 +1,86 @@
 import React from 'react';
 
+import { Link, useLocation } from 'react-router-dom';
+
 import './stylesheet.css';
 
 import TablePart from './components/TablePart';
 import ListPart from './components/ListPart';
 import TimestampPart from './components/TimestampPart';
 
+import { orgFileExtensions } from '../../../../lib/org_utils';
+
 import classNames from 'classnames';
 
 export default ({ parts, subPartDataAndHandlers }) => {
   let className;
 
+  let location = useLocation();
+
+  const renderLink = (part) => {
+    const id = part.get('id');
+    const uri = part.getIn(['contents', 'uri']);
+    const title = part.getIn(['contents', 'title']) || uri;
+    let target = uri;
+    if (uri.startsWith('file:')) {
+      target = uri.substr(5);
+      const isRelativeOrgFileLink =
```

Comment:
Rename this variable to `isRelativeLink` because it can also be a directory?

Reply:
Good catch! Fixed.